Gruntfile.js

```diff
@@ -310,6 +340,17 @@ module.exports = function(grunt) {
         console.log(`${targetName} extension is in ${dropExtensionPath}`);
     });
 
+    grunt.registerTask('package', function() {
```

Comment:
`'package'` sounds too generic, should we include `'report'` in the name of this task (an related code)?

Reply:
Or maybe `reporter-package`? Or `report-package`?

Reply:
Probably so. How about reporter? Or maybe reporter-package? Or report-package?

Reply:
`report-package` seems to cover purpose of the new package and let the door open to any other package we may want to introduce in the future.

Reply:
I changed it to `package-report`. What do you think?